ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -247,6 +293,17 @@ objects:
         fi
         cp $KUBECONFIG /tmp/admin.kubeconfig
         echo "Installation successful"
+        echo "Starting installer smoke tests..."
+        export SMOKE_KUBECONFIG=${KUBECONFIG}
+        export SMOKE_MANIFEST_PATHS=/tmp/cluster/generated/manifests
+        # 3 masters/3 workers/1 bootstrap(if post remove etcd)
+        # TODO: Make this a param
```

Comment:
What's this about?  Are you suggesting calculating this from `inputs.yaml`?

Reply:
yea, or keeping it in a variable rather than hardcoded there